jbmc/src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -217,7 +217,7 @@ void java_bytecode_languaget::set_language_options(const optionst &options)
                   << "object, it will be ignored." << messaget::eom;
       }
       else
-        static_values_file = std::move(to_json_object(tmp_json));
+        static_values_json = std::move(to_json_object(tmp_json));
```

Comment:
Aha, this does get done. Suggest squash into earlier commit that changes the type.